### PR TITLE
Feature/id 014 hero section

### PR DESCRIPTION
**Pull request by MeryemKh**

### CODE REVIEW

File: src/components/Hero/index.js [hunk 1]
```diff
@@ -1,9 +1,40 @@
...
+  <section className={styles.heroSection}>
+    {/* <div className = {styles.bckImg}>
+      <BackgroundImage fluid={backImg}> */}
+	      <h2>{headline}</h2> 
+        <p>{children}</p>
+        <div>
+          {/* <Link to ={button.link}>{button.text}</Link> */}
+          <button className={styles.button} >
+            <Link to = {button.link}>
+              <text className={styles.text}>{button.text}</text>
```
Comment (fraed) [inline src/components/Hero/index.js]: This seems not to be a valid tag.

File: src/components/Hero/styles.module.styl [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+  height 250px
+  line-height 25px
+  margin 20px
+  max-width 3000px
+  padding 40px 100px
+
+  .button
+    background-color primary-color
+    display inline-block
+    borderRadius 3px
```
Comment (fraed) [inline src/components/Hero/styles.module.styl]: Invalid style property

File: src/components/Hero/styles.module.styl [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+  --section-bg-color bg-color
+
 .hero-section
-  margin 0
+  background-color var(--section-bg-color)
+  height 250px
+  line-height 25px
+  margin 20px
+  max-width 3000px
+  padding 40px 100px
```
Comment (fraed) [inline src/components/Hero/styles.module.styl]: Please check this styling for mobile devices, too.

File: src/components/Hero/styles.module.styl [hunk 4]
```diff
@@ -1,5 +1,29 @@
...
 
+:root
+  --section-bg-color bg-color
+
 .hero-section
-  margin 0
+  background-color var(--section-bg-color)
+  height 250px
+  line-height 25px
+  margin 20px
```
Comment (fraed) [inline src/components/Hero/styles.module.styl]: Let's get rid of these margins.

**Comment by fraed** [comment]
I changed the PR Base to develop and now we have conflicts to be resolved before merging. Could you please merge in develop and resolve the conflicts?

Please check your styles. A lot of selectors are invalid. You can check them via Dev-Tools. I gave one example in the Files changed area.

Please check the component for mobile devices.

Please format your code with the command `yarn format`

Afterwards I am going to make a suggestion for the background image.

File: src/components/Hero/index.js (hunk 1 above)
Comment (MeryemKh) [reply src/components/Hero/index.js]: Do you mean the <text> tag?

Comment (MeryemKh) [reply src/components/Hero/index.js]: Do you mean the `text` tag?

**Pull request merged**